browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -29,6 +30,10 @@ namespace brave {
 
 namespace {
 
+const std::vector<std::string> query_string_trackers = {
```

Comment:
Such global objects are prohibited by CC. Please check examples at `no_destructor.h`

Reply:
That's a really neat wrapper, thanks!